fluentui_core/src/main/java/com/microsoft/fluentui/compose/ModalPopup.kt

```diff
@@ -122,18 +152,32 @@ fun convertWindowInsetsCompatTypeToWindowInsets(windowInsetsCompatType: Int): Wi
 
 /** Custom compose view for [BottomDrawer] */
 private class ModalWindow(
-    private var onDismissRequest: () -> Unit,
+    private var onDismissRequest: (() -> Unit)? = null,
+    private var properties: PopupProperties,
     private val composeView: View,
+    density: Density,
     saveId: UUID,
+    private val popupLayoutHelper: PopupLayoutHelper = if (Build.VERSION.SDK_INT >= 29) {
```

Comment:
can you add a comment explaining, why are are using 2 different versions and what's different in these versions

Reply:
Will add this